regtests/client/python/cli/polaris_cli.py

```diff
@@ -36,9 +37,12 @@ class PolarisCli:
     available in the Python client API.
 
     Example usage:
-    * ./polaris --client-id ${id} --client-secret ${secret} --host ${hostname} principals create example_user
-    * ./polaris --client-id ${id} --client-secret ${secret} --host ${hostname} principal-roles create example_role
-    * ./polaris --client-id ${id} --client-secret ${secret} --host ${hostname} catalog-roles list
+    * ./polaris --client-id ${id} --client-secret ${secret} --host ${hostname} --port ${port} principals create example_user
```

Comment:
I would rather not more than double the number of examples here; there are further examples within each command

Reply:
Ah, this was a typo, was only supposed to add a single example. Fixed.